recipes/ccache-toolchain/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "ccache-toolchain" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - ccache      # [unix]
+  run:
+    - ccache      # [unix]
+    - toolchain 2.*
+
+test:
+  commands:
+    # Verify the scripts are in-place.
+    {% for state in ["activate", "deactivate"] %}
+    - test -f "${PREFIX}/etc/conda/{{ state }}.d/ccache_toolchain_{{ state }}.sh"  # [unix]
+    {% endfor %}
+    {% for cc in ["gcc", "g++", "clang", "clang++", "gfortran"] %}
```

Comment:
It would be nice to add some test executing the symlinks. I think just a running them with ```--help``` would be enough